src/ggml.c

```diff
@@ -5144,9 +5144,9 @@ struct ggml_tensor * ggml_acc_impl(
 
     result->op   = GGML_OP_ACC;
     result->grad = is_node ? ggml_dup_tensor(ctx, result) : NULL;
-    result->src0 = a;
-    result->src1 = b;
-    result->opt[0] = c;
+    result->src[0] = a;
+    result->src[1] = b;
+    result->src[0] = c;
```

Comment:
why are we re-writing to 0th index here again? perhaps a mistake? This should be src[2]?

Reply:
additionally, do we need a bounds check here?

Reply:
> why are we re-writing to 0th index here again? perhaps a mistake? This should be src[2]?

Yes, I'm fixing all indices as soon as possible :).